schemas/latest/ide-targeted/plugin-overrides.json

```diff
@@ -414,6 +422,123 @@
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "markdownDescription": "The arguments to supply to the dockerfile build."
+                  },
+                  "buildContext": {
+                    "description": "Path of source directory to establish build context. Defaults to ${PROJECT_ROOT} in the container",
+                    "type": "string",
+                    "markdownDescription": "Path of source directory to establish build context. Defaults to ${PROJECT_ROOT} in the container"
+                  },
+                  "git": {
+                    "description": "Project's Git source",
+                    "type": "object",
+                    "properties": {
+                      "checkoutFrom": {
+                        "description": "Defines from what the project should be checked out. Required if there are more than one remote configured",
+                        "type": "object",
+                        "properties": {
+                          "remote": {
+                            "description": "The remote name should be used as init. Required if there are more than one remote configured",
+                            "type": "string",
+                            "markdownDescription": "The remote name should be used as init. Required if there are more than one remote configured"
+                          },
+                          "revision": {
+                            "description": "The revision to checkout from. Should be branch name, tag or commit id. Default branch is used if missing or specified revision is not found.",
+                            "type": "string",
+                            "markdownDescription": "The revision to checkout from. Should be branch name, tag or commit id. Default branch is used if missing or specified revision is not found."
+                          }
+                        },
+                        "additionalProperties": false,
+                        "markdownDescription": "Defines from what the project should be checked out. Required if there are more than one remote configured"
+                      },
+                      "remotes": {
+                        "description": "The remotes map which should be initialized in the git project. Must have at least one remote configured",
+                        "type": "object",
+                        "additionalProperties": {
+                          "type": "string"
+                        },
+                        "markdownDescription": "The remotes map which should be initialized in the git project. Must have at least one remote configured"
+                      }
+                    },
+                    "additionalProperties": false,
+                    "markdownDescription": "Project's Git source"
+                  },
+                  "gitLocation": {
```

Comment:
I would say it should be a part for dockerfileGitSource instead of being on the top level, but I may miss something.